src/vcf/fixer.cpp

```diff
@@ -14,398 +14,585 @@
  * limitations under the License.
  */
 
+#include <set>
+
 #include "vcf/fixer.hpp"
 #include "vcf/string_constants.hpp"
 
 namespace ebi
 {
-    namespace vcf
-    {
-        void Fixer::fix(size_t line_number, std::vector<char> &line, Error &error)
-        {
-            this->line_number = line_number;
-            this->line = &line; // this will be valid until this function returns
-            error.apply_visitor(*this);
-            this->line = nullptr;
-        }
+  namespace vcf
```

Comment:
Would it be possible to revert the indentation style to the previous one, as a lot of changes seem to be reported just because of the 4 vs 2 spaces. It is hard for me to see the actual changes, possibly because I was away while this was developed.

Edit: I just saw [@jmmut comment](https://github.com/EBIvariation/vcf-validator/pull/78/commits/cb95994c15fdc2b62ee73598096de8592110d283#r127690791). What do you both think about creating another PR, after this one is merged, to unify the indentation across all the source files in the project?